package.json

```diff
@@ -7,6 +7,9 @@
     "datocms-plugin",
     "instagram"
   ],
+  "files": [
+    "build",
```

Comment:
Hi @stefanoverna , I think the trailing comma needs removing

Reply:
yes, true! 👍🏻